index.html

```diff
@@ -60,21 +60,22 @@
 
     <div class="container">
       <!-- Example row of columns -->
+      <div class="home-Page">
       <div class="row">
         <div class="col-md-4">
-          <h2>Heading</h2>
-          <p>Donec id elit non mi porta gravida at eget metus. Fusce dapibus, tellus ac cursus commodo, tortor mauris condimentum nibh, ut fermentum massa justo sit amet risus. Etiam porta sem malesuada magna mollis euismod. Donec sed odio dui. </p>
-          <p><a class="btn btn-default" href="#" role="button">View details &raquo;</a></p>
+          <h2>Where Are the Poems?!</h2>
+          <p>Well you asked and you shall recieve. All the greatest poems ever in one space. It's pretty wicked sweet man!</p>

```

Comment:
It's nice that the buttons actually do stuff!  And that there are actual descriptions in place.  They don't look quite as word salad-y as lorem ipsum does, either.